infrastructure/run_deployment

```diff
@@ -44,18 +44,16 @@ ansible-vault view --vault-password-file="$VAULT_PASSWORD_FILE" ansible_ssh_key.
 
 function main() {
 
-downloadLobbyJar
+  downloadHttpServerJar
 
-downloadHttpServerJar
+  downloadBotJar
 
-downloadBotJar
+  downloadMigrations
```

Comment:
Shouldn't this pr remove the old functions as well?

Reply:
Good call, yes it should. Updated: 54db2ceafa55d57161b49bcd78527b87ce60ad7c